### DISCUSSION TITLE
MikroTik cAP ac, RouterOS 7.15.3, `wifi-qcom-ac` package

### DISCUSSION THREAD

**Discussion by Zoxc** [Zoxc/crusader#45] [Wi-Fi routers and access points]
This is a test of the MikroTik cAP ac (RBcAPGi-5acD2nD) running RouterOS 7.15.3 using the `wifi-qcom-ac` package. It is configured as an access point. The client used is an Intel AX210 on Windows 10. The download direction is from the AP.

It suffers from non-deterministic packet loss of the measurement packets.

### 2.4 Ghz, 20 Mhz, RSSI -58 dBm:
![2g 20m](https://github.com/user-attachments/assets/f3c58578-7c0d-4ce5-ab3f-7fb3f8699356)

### 5 Ghz, 20 Mhz, RSSI -69 dBm:
![5g 20m](https://github.com/user-attachments/assets/cdc16308-4b93-4178-b383-151a556dd2a6)

### 5 Ghz, 40 Mhz, RSSI -69 dBm:
![5g 40m](https://github.com/user-attachments/assets/661434a2-b9ad-452f-9859-a0eaa820a32d)

### 5 Ghz, 80 Mhz, RSSI -69 dBm:
![5g 80m](https://github.com/user-attachments/assets/03a01e1b-255b-404c-9599-9a978baa8490)

**Comment by Zoxc** [comment]
As a bonus, here's a wired benchmark with it configured as a router. The CPU running at 896 Mhz. CAKE configured with 1 Gbps shapers on both WAN and LAN.

![1s v4 cake 896 mhz](https://github.com/user-attachments/assets/0e25189b-ae33-4d44-a739-e3c43ec7edf2)

**Comment by dtaht** [comment]
in this case you look to have run out of CPU too shape the up + down

**Reply by Zoxc** [reply]
That doesn't seem to be the case as it's a quad core and a single core manages 710 Mbps. Up and down should be able to be handled by separate cores, but that doesn't seem to work out well for some reason. I suspect the `auto` IRQ function in RouterOS, though there might be something else going on.
